meteor-app/imports/api/members/methods.js

```diff
@@ -11,5 +11,44 @@ Meteor.methods({
       log.error({ e })
       throw new Meteor.Error(500, e.sanitizedError.reason)
     }
-  }
+  },
+  'members.remove'(id) {
+    try {
+      log.info('removing member id: ', id)
+      return Members.remove({ _id: id })
+    } catch (e) {
+      log.error({ e })
+      throw new Meteor.Error(500, e.sanitizedError.reason)
+    }
+  },
+  'members.setPin'(id, pin) {
+    try {
+      log.info('setting pin: ', id, pin)
+      // return Members.remove({ _id: id })
+      return Members.update({ _id: id }, { $set: { pin: pin } })
+    } catch (e) {
+      log.error({ e })
+      throw new Meteor.Error(500, e.sanitizedError.reason)
+    }
+  },
+  'members.forgotPin'(id, method, destination) {
+    log.info(`sending pin for member ${id} via ${method} to ${destination}`)
+    try {
+      // make DB query and grab the pin.
+      const member = Members.findOne(id)
+      const pin = member.pin
+      // construct message.
+      const message = 'Your PIN for Back2Bikes attendance app is: \n ' + pin
+      if (method == 'email') {
+        debug('sending PIN reminder via email.', destination, message)
+        return Meteor.call('sendEmail', destination, message, 'Back2Bikes Pin Reminder')
```

Comment:
You are already in a server method here, so there is no need to do a Meteor.call for another server method. You can just export the function, import the module here and call it. Then you don't need to deal with the asynchronicity of method calls.